IngerML/lab_2/main.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <cstdlib>
+#include <ctime> 
+using namespace std;
+const int razm = 10;
+int main()
+{
+	int i;
+	cout << "Mass 1:";
+	srand(time(0));
+	{
+	int arr[razm];
+	for (i = 0; i < razm; i++)
+	{
+		arr[i] = rand() % 100;
+		cout << arr[i] << ' ';
+	}
+	cout << endl;
+	int T = 0;
+	int a = 0;
+	for (int i = 1; i < razm; i++)
+	{
+		T = arr[i];
+		a = i - 1;
+		while ((a >= 0) && (arr[a] > T))
+		{
+			arr[a + 1] = arr[a];
+			a = a - 1;
+			arr[a + 1] = T;
+		}
+	}
+	cout << "Mass 2:";
+	for (i = 0; i < razm; i++) { cout << arr[i] << ' ';	}
+	cout << endl;
+
+	int min = 101;
+	int max = 0;
+	int sum = 0;
+	cout << "Mass 3:";
+	for (i = 0; i < razm; i++)
+	{
+		if (arr[i] % 2 == 1) { cout << arr[i] << " "; }
+	}
+	cout << endl;
+
+	cout << "Control numbers:" << endl;
+	for (i = 0; i < razm; i++)
+	{
+		if (arr[i] > max) { max = arr[i]; } 
+		if (arr[i] < min) { min = arr[i]; }
+		sum = sum + arr[i];
+	}
+	cout << "MAX: " << max << endl;
+	cout << "MIN: " << min << endl;
+	cout << "SRED: " << 0.1 * sum << endl;

```

Comment:
You are using numericliteral 0.1 to find the average and that is incorrect. What if i will change the value of "razm" to 11?